modules/end-user-guide/pages/selecting-a-workspace-ide.adoc

```diff
@@ -7,7 +7,7 @@
 [id="selecting-a-workspace-ide"]
 = Selecting a workspace IDE
 
-The default in-browser IDE in a new workspace is link:https://github.com/che-incubator/che-code[Microsoft Visual Studio Code - Open Source]. In addition, link:https://github.com/che-incubator/jetbrains-editor-images[Community Editions of JetBrains IntelliJ IDEA and JetBrains PyCharm] are available as link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] features. link:https://theia-ide.org/docs/[Eclipse Theia] is deprecated and will be removed in the next release.
```

Comment:
Technology Preview is mentioned in the downstream docs.
I'd like to simplify this paragraph and keep all IDE links and descriptions to a single source that is the table file. I'm concerned that we don't eventually end up with conflicting info (some updated and some not).